### ISSUE TITLE
design: dark mode's contrast in Project

### ISSUE THREAD

**Issue by Soliine** [ICIJ/datashare#1700]
Context:
Dark mode > Projects > One project's insights > On text Created and Updated, #6F6F6F on #0F0F10 is not compliant with WCAG AA. 

Request:
It should be in secondary-body-color (#6F6F6F) in Light mode and should be #BFC1C4 in Dark Mode to comply with WCAG AA.

**Comment by pirhoo** [comment]
In fact we already have an alternating color for this called "secondary": 

* `#c7c9cd` on dark
* `#212529` on light

Both are compliant with WCAG AA. Would that work for you?

![Image](https://github.com/user-attachments/assets/b60b4fda-2b82-46ec-b52f-96c8deace910)
![Image](https://github.com/user-attachments/assets/38341289-7cbb-4f3b-9f82-98b6c8a5252e)

**Comment by Soliine** [comment]
#212529 on light looks very darker on Figma than what you show here, are you sure about this?

It looks like this:
<img width="332" alt="Image" src="https://github.com/user-attachments/assets/7f06481e-c139-4b35-a337-73f0b8bcc37f" />

**Comment by pirhoo** [comment]
Ah yes sorry, it's #212529 but with an alpha of 0.75:

https://github.com/twbs/bootstrap/blob/v5.3.3/scss/_variables.scss#L441

We can tweak it of course!

**Comment by pirhoo** [comment]
Should we close the issue then?
